packages/web/config/chain-infos.ts

```diff
@@ -44,8 +44,8 @@ const chainInfos = (
           isFeeCurrency: true,
           gasPriceStep: {
             low: 0,
```

Comment:
```suggestion
            low: 0.0025,
```
the minimum gas price should be 0.0025 after v15, so low gas price is also should be 0.0025

Reply:
what's the ETA on the upgrade?